groovy/postInit/chemistry/organic_chemistry/polymers/PolyvinylChain.groovy

```diff
@@ -26,42 +29,24 @@ TBR.recipeBuilder()
     .EUt(30)
     .buildAndRegister()
 
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('chlorine') * 100)
-    .fluidInputs(fluid('ethylene') * 50)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('hydrogen_chloride') * 50)
-    .duration(5)
-    .EUt(30)
-    .buildAndRegister()
-
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('ethylene') * 50)
-    .fluidInputs(fluid('hydrogen_chloride') * 50)
-    .fluidInputs(fluid('oxygen') * 50)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('water') * 50)
-    .duration(10)
+// Dehydrochlorination
+PYROLYSE_OVEN.recipeBuilder()
+    .fluidInputs(fluid('one_two_dichloroethane') * 1000)
+    .fluidOutputs(fluid('cracked_one_two_dichloroethane_mix') * 2000)
+    .duration(80)
     .EUt(30)
     .buildAndRegister()
 
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('ethane') * 50)
-    .fluidInputs(fluid('chlorine') * 200)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('hydrogen_chloride') * 150)
-    .duration(10)
-    .EUt(30)
-    .buildAndRegister()
-
-CSTR.recipeBuilder()
-    .fluidInputs(fluid('ethane') * 50)
-    .fluidInputs(fluid('hydrogen_chloride') * 50)
-    .fluidInputs(fluid('oxygen') * 100)
-    .fluidOutputs(fluid('vinyl_chloride') * 50)
-    .fluidOutputs(fluid('water') * 100)
-    .duration(10)
-    .EUt(30)
+DISTILLATION_TOWER.recipeBuilder()
+    .fluidInputs(fluid('cracked_one_two_dichloroethane_mix') * 2000)
+    .fluidOutputs(fluid('hydrogen_chloride') * 1000)
+    .fluidOutputs(fluid('vinyl_chloride') * 500)
+    .fluidOutputs(fluid('acetylene') * 200)
+    .fluidOutputs(fluid('one_one_dichloroethane') * 200)
+    .fluidOutputs(fluid('ethylene') * 50)
+    .fluidOutputs(fluid('one_two_dichloroethylene') * 50)
+    .duration(80)
+    .EUt(120)
     .buildAndRegister()
```

Comment:
what are the source for these ratios?
if you have the source can you comment it in the code

Reply:
The rates are arbitary, although based on this paper
https://koreascience.kr/article/JAKO200211920879285.pdf
![image](https://github.com/user-attachments/assets/d2711306-e55e-41de-8d0a-b10858234bc5)

Reply:
comment the source for this recipe then

Reply:
in PolyvinylChain.groovy

Reply:
don